api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -796,6 +796,215 @@
 			<column name="uuid" type="char(38)" />
 		</addColumn>
 	</changeSet>
+	<changeSet author="vshankar" id="201609171146-1">
+		<preConditions onFail="MARK_RAN">
+			<foreignKeyConstraintExists foreignKeyName="member_patient"/>
+		</preConditions>
+		<dropForeignKeyConstraint constraintName="member_patient" baseTableName="cohort_member"/>
+	</changeSet>
+	<changeSet author="vshankar" id="201609171146-1.1">
+		<preConditions onFail="MARK_RAN">
+			<foreignKeyConstraintExists foreignKeyName="parent_cohort"/>
+		</preConditions>
+		<dropForeignKeyConstraint constraintName="parent_cohort" baseTableName="cohort_member"/>
+	</changeSet>
+	<changeSet author="vshankar" id="201609171146-1.2">
+		<preConditions onFail="MARK_RAN">
+			<primaryKeyExists tableName="cohort_member"/>
+		</preConditions>
+		<comment>Dropping primary key for cohort_member</comment>
+		<dropPrimaryKey tableName="cohort_member"/>
+	</changeSet>
+	<changeSet author="vshankar" id="201609171146-2">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="cohort_member" columnName="cohort_member_id"/>
+			</not>
+		</preConditions>
+		<addColumn tableName="cohort_member">
+			<column name="cohort_member_id" type="int" autoIncrement="true">
```

Comment:
Won't this fail for existing rows since they wont have values for this column? If you tested and it passed it's fine